sklearn/metrics/ranking.py

```diff
@@ -251,18 +255,28 @@ def roc_auc_score(y_true, y_score, average="macro", sample_weight=None):
     0.75
 
     """
-    def _binary_roc_auc_score(y_true, y_score, sample_weight=None):
+    def _binary_roc_auc_score(y_true, y_score, sample_weight=None,
+                              pos_label=None):
         if len(np.unique(y_true)) != 2:
             raise ValueError("Only one class present in y_true. ROC AUC score "
                              "is not defined in that case.")
 
         fpr, tpr, tresholds = roc_curve(y_true, y_score,
+                                        pos_label=pos_label,
                                         sample_weight=sample_weight)
         return auc(fpr, tpr, reorder=True)
 
-    return _average_binary_score(
-        _binary_roc_auc_score, y_true, y_score, average,
-        sample_weight=sample_weight)
+    y_type = type_of_target(y_true)
+    if y_type == "binary":
+        _partial_binary_roc_auc_score = partial(_binary_roc_auc_score,
+                                                pos_label=pos_label)
+        return _average_binary_score(
+            _partial_binary_roc_auc_score, y_true, y_score, average,
+            sample_weight=sample_weight)
+    else:
+        return _average_binary_score(
```

Comment:
What should we do with `pos_label` in this case?? Use it? Raise an error if it is set?